frontend/src/components/Navbar.vue

```diff
@@ -0,0 +1,49 @@
+<template>
```

Comment:
Great start. I think we should create a nav-items array of objects in the script portion that defines all the things we need, and then run  a v-for item in navItem (or something to that effect). OR just wrap all the elements with the router link, I see that some routes  are still anchor tags (<a>), both options involve making the boilerplate pages for the other routes just like you did for admin and home